### ISSUE TITLE
minor doc improvement: use example with multiple gems for 'filter_gems_from_backtrace'

### ISSUE THREAD

**Issue by e2** [rspec/rspec-core#1846]
#### TL;DR - I'm suggesting reworking `configuration/excluding-lines-from-the-backtrace` doc page

Relish app / cucumber scenario  for backtrace exclusion (https://www.relishapp.com/rspec/rspec-core/docs/configuration/excluding-lines-from-the-backtrace) mentions:

```ruby
config.filter_gems_from_backtrace "my_gem"
```

But I wasn't sure how to use multiple gems, so I guessed it was an array, expecting to get an error at least:

```ruby
config.filter_gems_from_backtrace %w(foo bar)
```
The resulting regexp was of course something like `/(...)[foo, bar](...)/`, so it silently didn't work (ignoring nothing).

The rdoc OTOH is more useful - by showing an example with parameters:

```ruby
config.filter_gems_from_backtrace "foo", "bar"
```

This is much nicer (very idiomatic to Ruby!), though I find it a bit surprising to pass an array as parameters (just uncommon in Ruby libs out there).

I thought about a PR for the cucumber scenario, but the example there doesn't make sense with multiple gems scenario:

```ruby
config.filter_gems_from_backtrace "my_gem", "another_gem"
```

And besides the minor issue of "another_gem" being "useless" in the context of the scenario ("mygem" is the only gem in the backtrace) - the example can be easily missed (it's at the bottom of the page).

At the same time, mentioning at the top of the page that `filter_gems_from_backtrace` expects gems as parameters (and not an array) seems too verbose (examples are better than descriptions).

Maybe I'm splitting hairs over this - though I've never tried (needed) and appreciated the feature as much as now (backtrace takes up way too much screen space) ...

... and IMHO needs more emphasis than the patterns:

```ruby
/\/lib\d*\/ruby\//,
/org\/jruby\//,
/bin\//,
/lib\/rspec\/(core|expectations|matchers|mocks)/
```

Seem like the main thing people are after (ignoring gems in backtrace) is obscured in the example.

So reworking the whole "backtrace-doc" page seems like the best option. Agreed? Thoughts?

**Comment by JonRowe** [comment]
I'd be ok with a PR tweaking the example, go for it.

**Comment by soulcutter** [comment]
The typical way to denote varargs in a method is by describing it with ellipses `filter_gems_from_backtrace(…)`

**Comment by myronmarston** [comment]
Feel free to submit a PR.  Our docs are always a work in progress, so we always welcome improvements.  No promise that we'll merge everything, of course, but it's easier to discuss specific recommendations in the form of a PR making a concrete change than in an issue discussing it in the abstract.

**Comment by e2** [comment]
@myronmarston - good points.

Here's the PR: https://github.com/rspec/rspec-core/pull/1847

**Comment by myronmarston** [comment]
Closing in favor of #1847.
